app/services/registration_service.rb

```diff
@@ -0,0 +1,9 @@
+class RegistrationService
+  def from_invitation(email, invitation_token, user_data = {})
+    Invitation.find_by(email: email, token: invitation_token)
```

Comment:
I don't get it. Why do you need finding an invitation??
Are you programming an `if/else` working with `Exceptions`?